pkg/controller/ippool/controller_test.go

```diff
@@ -142,7 +144,7 @@ func TestHandler_OnChange(t *testing.T) {
 					Build(),
 				ipPool: newTestIPPoolBuilder().
 					Paused().
-					AgentPodRef("default", "default-ippool-1-agent").
+					AgentPodRef("default", "default-ippool-1-agent", testImageRepository+":"+testImageTag, "").
```

Comment:
NIT: I think you could declare two consts on the top, it might be convenient.

```
testImage = testImageRepository+":"+testImageTag
testImageNew = testImageRepository+":"+testImageTagNew
```

Reply:
Updated, thanks.